Dockerfile

```diff
@@ -93,5 +147,6 @@ RUN chown -R jovyan:jovyan /home/jovyan
 
 # Convert notebooks to the current format
 RUN find . -name '*.ipynb' -exec ipython nbconvert --to notebook {} --output {} \;
+RUN find . -name '*.ipynb' -exec ipython trust {} \;
```

Comment:
I actually preferred not having them trusted, since we pull in notebooks from other parties, including the strata-sv-2015-tutorial and @jvns's Pandas cookbook. It's fine for now.